styles.css

```diff
@@ -0,0 +1,43 @@
+html {
+	font-family: "Trebuchet MS";
+	box-sizing: border-box;
+}
+
+nav {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-between;
+	
+}
+
+nav div {
+	width: 15%;
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	justify-content: space-around;
+	color: black;
+}
+
+nav button {
+	width: 150px;
+	height: 30px;
+	border: 2px solid black;
+	background-color: white;
+	color: black;
+	font-size: 16px;
```

Comment:
could use rem units for font size and declare global font size of 10px at the top